### DISCUSSION TITLE
Is it possible to write a Training-Loop similar to Tensorflow with AutoKeras?

### DISCUSSION THREAD

**Discussion by krishnasrikard** [keras-team/autokeras#1576] [General]
I am interested in writing Training-Loop instead of ```.fit``` for Searching and Training the Model and the reason is to primarily access Gradients of the Model during Training...

Is there any possibility for that or atleast accessing Gradients?

Thanks.
